lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2078,6 +2079,25 @@ SubExprInfo ExprEmitter::visitSV(ArrayIndexInOutOp op) {
   return {Selection, arrayPrec.signedness};
 }
 
+SubExprInfo ExprEmitter::visitTypeOp(ArrayInjectOp op) {
+  os << '{';
+  auto arrayTy = op.getType().cast<ArrayType>();
+  for (unsigned i = 0, n = arrayTy.getSize(); i < n; ++i) {
```

Comment:
You should probably use array slicing for this. Arrays can get stupid long.

Something like: {a[10:4], replacement, a[2: 0]}.

Reply:
Awesome, thanks!